integration_tests/tests/utils.py

```diff
@@ -35,7 +39,9 @@ def make_token(username, secret_key, lifetime, claims):
     )
 
 
-query_kinds = ["daily_location", "modal_location", "flows"]
+query_kinds = list(FlowmachineQuerySchema.type_schemas.keys()) + list(
```

Comment:
I feel like the right-hand side is useful enough to add to flowmachine itself. Feel free to add to this PR, or we can create a separate issue for it.